src/entities/twin/api/hooks/useSearchV3.ts

```diff
@@ -11,23 +12,29 @@ export const useTwinSearchV3 = () => {
     async ({
       search,
       pagination = { pageIndex: 0, pageSize: 10 },
+      filters,
     }: {
-      search: string;
+      search?: string;
       pagination?: PaginationState;
-    }): Promise<{ data: Twin[]; pageCount: number }> => {
+      filters?: TwinFilters;
+    }): Promise<PagedResponse<Twin>> => {
```

Comment:
change to 
`}): Promise<PagedResponse<Twin_DETAILED>> => {`

additionally update in `src/entities/twin/api/types.ts`

```ts
export type Twin = components["schemas"]["TwinV2"];
export type Twin_SHORT = RequireFields<
  Twin,
  | "id"
  | "name"
  | "assignerUserId"
  | "authorUserId"
  | "markerIdList"
  | "statusId"
  | "tagIdList"
  | "twinClassId"
>;
export type Twin_DETAILED = RequireFields<
  Twin_SHORT,
  "createdAt" | "headTwinId"
>;
```

and in `src/entities/twin/libs/helpers.ts`

```ts
export const hydrateTwinFromMap = (
  twinDTO: Twin,
  relatedObjects?: RelatedObjects
): Twin_DETAILED => {
  const twin: Twin_DETAILED = Object.assign({}, twinDTO) as Twin_DETAILED;

  if (!relatedObjects?.twinClassMap) return twin;

  if (twinDTO.statusId && relatedObjects.statusMap) {
    twin.status = relatedObjects.statusMap[twinDTO.statusId] as TwinStatus;
  }

  if (twinDTO.twinClassId) {
    twin.twinClass = relatedObjects.twinClassMap[
      twinDTO.twinClassId
    ] as TwinClass_DETAILED;
  }

  if (twinDTO.authorUserId && relatedObjects.userMap) {
    twin.authorUser = relatedObjects.userMap[twinDTO.authorUserId] as User;
  }

  if (twinDTO.assignerUserId && relatedObjects.userMap) {
    twin.assignerUser = relatedObjects.userMap[twinDTO.assignerUserId] as User;
  }

  return twin;
};
```